docs/node.md

```diff
@@ -4,4 +4,141 @@ title: Setup full node
 sidebar_label: Setup full node
 ---
 
-TBD
\ No newline at end of file
+## Startup
+
+To start the node, run its executable:
+
+```bash
+radicle-registry-node
+```
+
+The node must store data on your hard drive.
+You can override the default location with a parameter:
+
+```bash
+radicle-registry-node --data-path <PATH>
+```
+
+### Connecting to the network
+
+The node connects by default to the FFNet network.
+It has a list of known nodes, which can be called to join the network for the first time.
+You can use your own list of bootstrap nodes with a startup parameter:
+
+```bash
+radicle-registry-node --bootnodes <ADDR>
```

Comment:
We should explain what `ADDR` is and link to more information. I believe it’s a libp2p multi address. We should also update the help text for this option.

Reply:
Fixed